lib/src/modules/renameInput/RenameInputForm.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+export function RenameInputForm({
+  stopEditing,
+  onRename,
+  label,
+  text,
+  id,
+  multiline,
+  classes,
+  placeholder,
+  formClass
+}) {
+  const [value, setValue] = useState(text);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (value.trim() !== '') {
+      onRename(value);
+    }
+    stopEditing();
+  };
+
+  const handleOnChange = e => {
+    setValue(e.target.value);
+  };
+
+  const handleEsc = e => {
+    const isEscKey = e.keyCode === 27;
+    const isEnterKey = e.keyCode === 13;
+    if (isEscKey) {
+      stopEditing();
+    }
+    if (isEnterKey) {
+      handleSubmit(e);
+    }
+  };
+
+  const InputType = multiline ? 'textarea' : 'input';
+  return (
+    <form onSubmit={handleSubmit} className={`rename-input-form ${formClass}`}>
+      <label htmlFor={id} className="hidden">
+        {label}
+      </label>
+      <div className={`rename-input-size-helper ${classes}`} aria-hidden="true">
```

Comment:
This is a stroke of genius 🙏

Reply:
💚 My only concern with this PR is that without your comment, I might not have known that this div is the magic that resizes the parent. Maybe if it was extracted to a component like `<InvisibleParentSizer />` or something like that (probably better named)